react-components/table-intro/step1.md

```diff
@@ -7,7 +7,7 @@ The following is an example of what the table component structure looks like:
 &lt;/Table&gt;
 </pre>
 
-Columns can be expressed as an array of strings, or an array of objects which match the type "ICell". The columns definition is passed to the `cells` prop of the Table component which can take the form of `Array<ICell | string>`. Below is what an ICell looks like.
+Columns can be expressed as an array of strings or an array of objects which match the type "ICell". An "ICell" is a TypeScript interface that represents a cell of the `<Table>`. The columns definition is passed to the `cells` property of the `<Table>` component which can take the form of `Array<ICell | string>`. Below is what the ICell interface looks like.
```

Comment:
`An "ICell is a TS interface......` still feels a little like a concept outside of PatternFly, how about: "ICell" is the name given to the Typescript interface that represents a cell of the `<Table>` . Otherwise reads much better.

I wonder if ICell should be wrapped in back-ticks or in quotation marks here? It is referring to code, but also to the name, but typically i think we have gone with wrapping it as code.